db/migrate/20170908160100_add_ip_address_and_user_agent_id_to_event_logs.rb

```diff
@@ -5,7 +5,7 @@ def self.up
     Lhm.cleanup(:run)
     Lhm.change_table :event_logs do |m|
       m.add_column :ip_address, "BIGINT"
-      m.add_column :user_agent_id, "BIGINT"
+      m.add_column :user_agent_id, "INT"
```

Comment:
How come we've been editing a past migration?

Reply:
This migration will fail under rails 5 - the default types for id fields have been changed to BIGINTS so the field types are mismatched. I can avoid editing it if you'd prefer that, but it will mean the migration is broken from Rails 5 onwards

Reply:
IMO, we should change this via a new migration and also drop old migrations from the repository (don't see any value in having them in here if they have hit production already).

Reply:
In general I agree with removing old migrations, but in this case we have an AWS env that's not being deployed to as often so might need these migrations for longer than usual.  Also I think signon is special in that it doesn't get restored from staging/prod over night so we might have to do a thing to make sure the migration change actually goes through properly.